test/phoenix_live_view/diff_test.exs

```diff
@@ -590,7 +591,7 @@ defmodule Phoenix.LiveView.DiffTest do
       assert {MyComponent, "hello", _, _, _} = cid_to_component[0]
 
       assert_received {:mount, %Socket{endpoint: __MODULE__, assigns: assigns}}
-                      when assigns == %{flash: %{}, myself: 0}
+                      when assigns == %{flash: %{}, myself: %CID{cid: 0}}
```

Comment:
Not sure how you feel about this. Figured I'd use the full module name in most places for clarity, but it got a bit noisy in this test.